src/pages/listing/index.js

```diff
@@ -210,6 +210,9 @@ export function controller() {
 
         if(window.confirm("Remove " + data.name + "?")) {
             ref.remove().catch(console.error.bind(console));
+
+            // Can't figure out why `ref.remove()` was redirecting to "/"
```

Comment:
Okay. First up, you're right. That is exactly where the problem was. 

But it didn't occur to me, because I didn't think that event listener would be listening while I was on the listing page. I wasn't even thinking about stuff in `content-edit/`. But I just ran a quick test, and proved it out. The behavior only occurs if you've been to a `content-edit` page and haven't refreshed since then.

As a best practice, it looks like we should be calling `ref.off()` before we set up any `listing` or `content-edit` pages. Another quick test showed me that going from listing->entry 4 times adds 4 separate instances of every `ref.on("value", callback);` listener, so we're hamstringing our performance and doing a lot of extra rapidfire work.